terraform/anomaly_detection/README.md

```diff
@@ -0,0 +1,83 @@
+# GenAI & ML inference project deployment
+
+This directory contains the Terraform code to spawn a Google Cloud project
+with all the necessary infrastructure and configuration required for running
+the GenAI & ML inference solution guide.
+
+These deployment scripts are part of the
+[Dataflow Gen AI & ML solution guide](../../use_cases/GenAI_ML.md).
+
+## Bill of resources created by this script
+
+The scripts will create the following resources
+
+| Resource             |         Name          | Description                                                                                                                                                                                                                                                                                      |
```

Comment:
Update this table with the actual resources created by the TF code. I am missing the Bigtable and BigQuery details here.